torchchat/cli/convert_hf_checkpoint.py

```diff
@@ -21,9 +24,176 @@
 
 from torchchat.model import ModelArgs
 
+def remap_llava_checkpoint(llava_ckpt):
```

Comment:
Was this written inhouse?

Reply:
I'm not pretty following your question. 
This function is consumed by `convert_llava_checkpoint` to get remapped checkpoint.
I made this as an individual function to simply the logic